gTea/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
```

Comment:
If I understand correctly, this can be added to the .gitignore file, unless its relevant for other developers.

Reply:
Just checked, _.idea_ folder is listed in .gitignore. I can remove it from history, though.

Reply:
No then it's fine.